ApplicationLibCode/CMakeLists.txt

```diff
@@ -49,6 +50,7 @@ if(Qt5Core_FOUND)
       Qt5::Concurrent
       Qt5::PrintSupport
       Qt5::Svg
+      Qt5::Sql
```

Comment:
Need to copy the sqlite SQL plugin on install also?

Reply:
Not required on Windows, did adjustments on Linux